js/menu.js

```diff
@@ -0,0 +1,25 @@
+const html = document.getElementsByTagName('html')[0];
+const navMenu = document.querySelector('.desktop-nav');
+const showBtn = document.querySelector('nav .menu-bars');
+const closedBtn = document.querySelector('nav .closed-menu-bars');
+const navLinks = document.querySelectorAll('nav ul li');
+
+function showMenu() {
+  navMenu.classList.add('open');
+  closedBtn.style.display = 'block';
+  html.style.overflow = 'hidden';
+}
+
+function closeMenu() {
+  navMenu.classList.remove('open');
+  closedBtn.style.display = 'nonne';
+  html.style.overflow = 'auto';
+  console.log(navMenu);
+}
```

Comment:
Also here, please kindly adhere to the suggestion that is stated by the previous reviewer, and you are good to go 👍🏾

Reply:
Also here, kindly adhere to the suggestion made by the previous reviewer about committing `console.log()`.  I think it would be best if you deleted it, and you are good to go 👍🏾